app/src/main/java/org/kiwix/kiwixmobile/intro/IntroActivity.kt

```diff
@@ -0,0 +1,130 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.intro
+
+import android.os.Bundle
+import android.os.Handler
+import android.view.View
+import android.widget.ImageView
+import androidx.viewpager.widget.ViewPager
+import androidx.viewpager.widget.ViewPager.OnPageChangeListener
+import kotlinx.android.synthetic.main.activity_intro.*
+import org.kiwix.kiwixmobile.R
+import org.kiwix.kiwixmobile.core.Intents.internal
+import org.kiwix.kiwixmobile.core.base.BaseActivity
+import org.kiwix.kiwixmobile.core.di.components.CoreComponent
+import org.kiwix.kiwixmobile.core.main.CoreMainActivity
+import org.kiwix.kiwixmobile.kiwixActivityComponent
+import java.util.Timer
+import java.util.TimerTask
+import javax.inject.Inject
+
+class IntroActivity : BaseActivity(), IntroContract.View {
+
+  companion object {
+    private const val timerDelay: Long = 0
+    private const val timerPeriod: Long = 2000
+    private const val animationDuration: Long = 800
+  }
+
+  private val handler = Handler()
+  private val timer = Timer()
+
+  @Inject
+  internal lateinit var presenter: IntroContract.Presenter
+  private lateinit var airPlane: ImageView
+  private var currentPage = 0
+  private lateinit var views: Array<View>
+
+  @Suppress("EmptyFunctionBlock")
+  private val pageChangeListener: OnPageChangeListener = object : OnPageChangeListener {
+    override fun onPageScrolled(position: Int, positionOffset: Float, positionOffsetPixels: Int) {
+    }
+
+    override fun onPageSelected(position: Int) {
+      if (position == 1) {
+        airPlane.visibility = View.VISIBLE
+        airPlane.animate().translationX(airPlane.width.toFloat()).duration = animationDuration
+      } else {
+        airPlane.visibility = View.INVISIBLE
+        airPlane.animate().translationX(-airPlane.width.toFloat())
+      }
+      currentPage = position
+    }
+
+    override fun onPageScrollStateChanged(state: Int) {
+      if (state == ViewPager.SCROLL_STATE_DRAGGING) {
+        dismissAutoRotate()
+      }
+    }
+  }
+
+  override fun injection(coreComponent: CoreComponent) {
+    this.kiwixActivityComponent.inject(this)
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_intro)
+    get_started.setOnClickListener { startMainActivity() }
+    val layoutInflater = layoutInflater
```

Comment:
Remove this line

Reply:
Yes👍

Reply:
@abdulwd When i removed the line, encountered a build fail.